_posts/2020-02-07-rising-from-the-ashes.md

```diff
@@ -0,0 +1,46 @@
+---
+layout:     post
+title:      Rising from the Ashes
+date:       2020-02-07
+summary:    We've always enjoyed running incident response drills, but they were becoming stale. This post covers how we addressed the problems with our fire drills and iterated upon them
+author:     ols
+image:      firedrills/fire.jpg
+category:   Operations
+tags:       chaos engineering, fire drill
+---
+
+Just under two years ago, I wrote about [how we run fire drills (incident management role play scenarios) in the Core tribe](https://sbg.technology/2018/05/04/firedrills-in-core/). The post covered the mechanics of planning and running the drills, but also talked a bit about the tools that we were using. Since then, lots has changed in how our fire drills work, and this post will cover the what and why of those changes. It is not necessary to read that to understand the points in this article, but I would still recommend it so that you have additional context around what has changed.
+
+Practising your incident response is nothing new or special, and it's certainly not confined to the technology sector. I'm sure many of us remember fire drills at school. I was never really sure what good it was doing, but made the most of the time out of lessons, taking "walk calmly don't run" to a new level of slowness. It's this attitude that has attracted a lot of criticism of fire drills, the thought being that they breed complacency.
+
+## Complacency
+
+We found ourselves in this situation too, people were becoming complacent. Our fire drills that once had us revered within the company were delivering fewer and fewer positive outcomes, and they were seen as a chore rather than an exciting and useful pastime.
```

Comment:
```suggestion
We found ourselves in this situation too; people were becoming complacent. Our fire drills that previously saw us revered within the company were delivering fewer and fewer positive outcomes, and they were seen as a chore rather than an exciting and useful pastime.
```